packages/react-devtools-shared/src/devtools/views/ErrorBoundary/ErrorBoundary.js

```diff
@@ -141,10 +186,7 @@ export default class ErrorBoundary extends Component<Props, State> {
             dismissError={
               canDismissProp || canDismissState ? this._dismissError : null
             }
-            errorMessage={errorMessage}
-            isUnsupportedBridgeOperationError={
```

Comment:
This seems to be added by mistake since we do not have a `isUnsupportedBridgeOperationError` prop in ErrorView